src/component/modal/editRange/EditRangeModal.tsx

```diff
@@ -146,14 +146,14 @@ function EditRangeModal({
             multiplicity,
             coupling: '',
           };
-          if (hasCouplingConstant(multiplicity) && signal?.js) {
+          if (hasCouplingConstant(multiplicity) && signal?.js.length > 0) {
             js = { ...signal.js[counterJ] } as Coupling;
             js.coupling = Number(
               formatNumber(js.coupling, rangesPreferences.coupling.format),
             );
             counterJ++;
           }
-          js.multiplicity = translateMultiplet(js.multiplicity || '');
+          js.multiplicity = translateMultiplet(js.multiplicity || multiplicity);
```

Comment:
I am not sure to add the multilplicity from the splitter pattern. Automatically the signal.multilplicity is generated from js multiplicity values but if for any refactor the order of the signal.multiplicity differ from the order of js.multiplicity we will introduce an error. @lpatiny WTD?